src/applications/claims-status/tests/e2e/04.claim-ask-va.e2e.spec.js

```diff
@@ -38,23 +38,15 @@ module.exports = E2eHelpers.createE2eTest(client => {
 
   // click on link to page
   client
-    // I have no idea why this pause is required, but it sure is
-    .pause(2000)
     .click('.claims-alert-status a')
     .waitForElementPresent('.usa-button-secondary', Timeouts.normal)
     .axeCheck('.main');
 
-  // click on disabled button
-  client.click('.main .usa-button-primary').pause(500);
```

Comment:
Clicking on disabled elements will raise errors in this newer version of Nightwatch. Edited instead to ensure that the button is disabled.